### ISSUE TITLE
Slow on android

### ISSUE THREAD

**Issue by leocrawf** [shripalsoni04/nativescript-angular-drawer-template#2]
Each tap takes about 3 seconds every time before the view is changed on the screen. I tested this using a not so fast phone but still other nativescript drawer apps work faster for example the nativescript official theme app.

**Comment by shripalsoni04** [comment]
Currently this template uses `tns-core-modules` version 2.3.0, which has an issue of slow navigation in android. This has already been reported at https://github.com/NativeScript/NativeScript/issues/2735 and most probably resolved in latest release.

I will suggest you to follow below steps and check again:
1. `npm install tns-core-modules@next` - This will install latest release.
2. `tns platform remove android`
3. `tns platform add android`
4. `tns run android` 

Thanks.

**Comment by leocrawf** [comment]
Thank you so much . I will give it a try.

**Comment by leocrawf** [comment]
I am now getting an error after I when I click the menu button : "Cannot read property 'toggleDrawerState' of undefined in 'side-drawer-page.component.ts'

**Comment by shripalsoni04** [comment]
I have just created a [demo app](https://github.com/shripalsoni04/ns-ng-drawer-template-demo) which uses latest version of `tns-core-modules`. I tested it on both android and ios simulators and it is working without any issue.

I suggest you to test this demo app on your device/emulator.

If on demo app also, you are getting an issue let me know the platform (Android/iOS), version/API level of the platform and whether you are testing on emulator/device. The best thing that I can do is to test the demo app on the simulator of the device you are using.

Thanks.
